icu4c/source/common/util.h

```diff
@@ -185,6 +185,15 @@ class U_COMMON_API ICU_Utility /* not : public UObject because all methods are s
      */
     static int32_t parseInteger(const UnicodeString& rule, int32_t& pos, int32_t limit);
 
+    /**
+     * Parse an integer at pos using only ASCII digits.
+     * Base 10 only.
+     * @param pos INPUT-OUTPUT parameter.  On input, the first
+     * character to parse.  On output, the character after the last
+     * parsed character.
+     */
```

Comment:
offset (or position, or something), not character.
	     * @param pos INPUT-OUTPUT parameter.  On input, the offset
	     * of the first character to parse.  On output, the offset after the
	     * last parsed character.

Reply:
Done.  I copied and pasted it from the other function.  I changed it here and in the old function.

Reply:
Well, OK. There might not be a character after the last parsed character, so it is really just the index after the last parsed character. But this is okay for now.